### DISCUSSION TITLE
Livewire + refreshing responsible media works strange

### DISCUSSION THREAD

**Discussion by faxunil** [spatie/laravel-medialibrary#2634] [Q&A]
Hi!
I have MediaLibraryPro I made a banner component with Livewire what is displaying Media's responsive image.
With media in controller and in the blade: $banner->getFirstMedia('banner')
It works fine, but If make a wire:poll on it the random loaded images are not clear even looks as the 1px would be displayed.
If I load the page with refresh the same pictures are displaying ok. with the Livewire not.
Do You have any Ide what could be the problem?
Thanks
Gergely

**Comment by gabzon** [comment]
I found this issue, maybe it could help you

https://github.com/spatie/laravel-medialibrary/issues/2290

**Reply by faxunil** [reply]
Thanks, i will check
